src/components/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```

Comment:
- [ ] I think it would be better if the value of each action type follows the [duck's pattern](https://github.com/erikras/ducks-modular-redux) for the project's requirements. So it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`